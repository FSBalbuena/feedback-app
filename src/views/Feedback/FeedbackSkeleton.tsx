import Box from "@mui/material/Box";
import Typography from "@mui/material/Typography";
import Skeleton from "@mui/material/Skeleton";
import { FeedbackSkeleton as copyText } from "@/data/copy";

const FeedbackSkeleton = () => (
  <Box width={`90vw`} maxWidth={800}>
    <Typography variant="body1" gutterBottom>
      {copyText.LOADING_MESSAGE}
    </Typography>
    <Typography variant="h2" gutterBottom>
      <Skeleton width={"60%"} />
    </Typography>
    <Typography variant="body1" gutterBottom>
      <Skeleton />
    </Typography>
    <Typography variant="body1" gutterBottom>
      <Skeleton />
    </Typography>
    <Typography variant="body1" gutterBottom>
      <Skeleton />
    </Typography>
    <Typography variant="body1" gutterBottom>
      <Skeleton />
    </Typography>
    <Typography variant="body1" gutterBottom>
      <Skeleton />
    </Typography>
    <Typography variant="h3" gutterBottom>
      <Skeleton width={"60%"} />
    </Typography>
    <Typography variant="body1" gutterBottom>
      <Skeleton />
    </Typography>
    <Typography variant="body1" gutterBottom>
      <Skeleton />
    </Typography>
    <Typography variant="body1" gutterBottom>
      <Skeleton />
    </Typography>
  </Box>
);

export default FeedbackSkeleton;
