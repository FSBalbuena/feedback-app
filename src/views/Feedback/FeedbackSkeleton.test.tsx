import FeedbackSkeleton from "./FeedbackSkeleton";
import { render } from "@testing-library/react";

describe("FeedbackSkeleton", () => {
  describe("on Render", () => {
    it("should render", () => {
      const { container } = render(<FeedbackSkeleton />);
      expect(container).toMatchSnapshot();
    });
  });
});
