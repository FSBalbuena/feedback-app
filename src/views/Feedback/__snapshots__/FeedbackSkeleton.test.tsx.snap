// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeedbackSkeleton on Render should render 1`] = `
<div>
  <div
    class="MuiBox-root css-16el19j"
  >
    <p
      class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-vjrxpc-MuiTypography-root"
    >
       Creating your feedback advice
    </p>
    <h2
      class="MuiTypography-root MuiTypography-h2 MuiTypography-gutterBottom css-1lcbsct-MuiTypography-root"
    >
      <span
        class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-kkcbhd-MuiSkeleton-root"
        style="width: 60%;"
      />
    </h2>
    <p
      class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-vjrxpc-MuiTypography-root"
    >
      <span
        class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-kkcbhd-MuiSkeleton-root"
      />
    </p>
    <p
      class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-vjrxpc-MuiTypography-root"
    >
      <span
        class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-kkcbhd-MuiSkeleton-root"
      />
    </p>
    <p
      class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-vjrxpc-MuiTypography-root"
    >
      <span
        class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-kkcbhd-MuiSkeleton-root"
      />
    </p>
    <p
      class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-vjrxpc-MuiTypography-root"
    >
      <span
        class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-kkcbhd-MuiSkeleton-root"
      />
    </p>
    <p
      class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-vjrxpc-MuiTypography-root"
    >
      <span
        class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-kkcbhd-MuiSkeleton-root"
      />
    </p>
    <h3
      class="MuiTypography-root MuiTypography-h3 MuiTypography-gutterBottom css-uynfxu-MuiTypography-root"
    >
      <span
        class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-kkcbhd-MuiSkeleton-root"
        style="width: 60%;"
      />
    </h3>
    <p
      class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-vjrxpc-MuiTypography-root"
    >
      <span
        class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-kkcbhd-MuiSkeleton-root"
      />
    </p>
    <p
      class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-vjrxpc-MuiTypography-root"
    >
      <span
        class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-kkcbhd-MuiSkeleton-root"
      />
    </p>
    <p
      class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-vjrxpc-MuiTypography-root"
    >
      <span
        class="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-kkcbhd-MuiSkeleton-root"
      />
    </p>
  </div>
</div>
`;
